### ISSUE TITLE
Difference between Actor and Role

### ISSUE THREAD

**Issue by patagoniahiker** [optimajet/WorkflowEngine.NET#9]
Hi there are 2 very confusing items in your documentation page, Im hoping you can clarify it.

There is Actor settings, Name, Value and Rule:
-  Is this the same as a role in ASP MVC, or is it a user, how does correspond to the ASP MVC user. So why does it only get one rule, I thought a roles/actor can have a full collection of rules
- What  is the relationship or difference between Commands Actions and Transitions
- What is the diff between IWorkflowActionProvider and RuleProvider
- On serverside you have public ActionResult API(), but there where to get the change in state etc Post/ back to the controller

**Comment by dmelnikov** [comment]
Do you have a valid license with support?

**Comment by sergy79** [comment]
@patagoniahiker Just switch over to 

1. **Stateless** - fully free and very easy, then attach to jsplumb UI for free version, send me msg, I will email you some code to start.
2. **Nservice**bus, also free for 3 servers / company with full support for 3 servers

This is a poor choice, the documentation and service are booth very bad :-1:

**Comment by patagoniahiker** [comment]
@dmelnikov  I'm am trying the free license version, do I need something else to run that, am I missing something.

@sergy79 I like the stateless its very simple, how can I get the progress to the task, does the stateless object support that?

**Comment by dmelnikov** [comment]
@patagoniahiker I'm glad that you look this project. But you must understand - support isn't free even you're using a version with free limits.

@sergy79 please, don't mislead other users. WFE has well and fully documentation. This documentation is better than for "Stateless". And, please, don't tell about services if you didn't purchase those services. Thanks.

**Comment by patagoniahiker** [comment]
Its a simple question, your documentation did not explain it. Your own site says the free version is fine.
Your Tuple<> are over complicating, are very hard to follow or debug without documentation.

**Comment by dmelnikov** [comment]
The free version is fine for non-commercial and non-critical systems. It's truth.

A documentation can't give answers to any questions.
For this case, we offer the support service.

**Comment by sergy79** [comment]
you notice he removed some the documentation hahaha

dmelnikov has a horrible attitude :-1:
